routes/management/SignupsReport.ts

```diff
@@ -59,10 +71,10 @@ export default function signupsReportHandler(signups: SignupMetricsService, slac
 
       await slack.sendSignupsMessage(span, formattedEndDate, spanResult, allTimeResult);
       res.sendStatus(200);
-    } catch(err) {
-      (err as DevPortalError).action = `apply wrapup message`;
+    } catch (err: unknown) {
```

Comment:
I've noticed this above as well... can `err` be an `Error` since `DevPortalError` is extending `Error`?

Reply:
Apparently, TypeScript says "no." :( `Catch clause variable type annotation must be 'any' or 'unknown' if specified. ts(1196)`

I found a long issue on the topic as well: https://github.com/microsoft/TypeScript/issues/20024

Reply:
Interesting, did not know that!